clarify gRPC settings are for internal traffic (#1629)
pomerium/documentation: content/docs/reference/grpc.mdx

# gRPC Settings

> This reference covers Pomerium's **gRPC Settings** for internal Pomerium communication — between the different Pomerium services in [split service mode], or between the Enterprise Console and core Pomerium in a [Pomerium Enterprise](/docs/enterprise) deployment.

Reviewer: ```suggestion
Pomerium's gRPC settings apply to internal communication between:
- Pomerium services running in [split service mode].
- The Enterprise Console and core Pomerium in a [Pomerium Enterprise](/docs/enterprise) deployment.
```
